### ISSUE TITLE
Reduce to two pages as an extended abstract

### ISSUE THREAD

**Issue by JJ** [JJ/vizgec2016#2]
Which is the form it's been accepted.

**Comment by ccottap** [comment]
I will take a look. Shall we publish the whole version in arXiv? It may be useful as a reference in the extended abstract in which the interested reader can see the details expanded.

**Comment by JJ** [comment]
Yep, after a few corrections too. It's tagged so we can easily retrieve it
from the repo once we're done with this.

2016-04-28 13:16 GMT+02:00 Carlos Cotta notifications@github.com:

> I will take a look. Shall we publish the whole version in arXiv? It may be
> useful as a reference in the extended abstract in which the interested
> reader can see the details expanded.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/JJ/vizgec2016/issues/2#issuecomment-215392014

## 

JJ

**Comment by ccottap** [comment]
After a cutting spree, it's almost there. I suggest leaving the whole paper as the sole reference. With that, it's in two pages.
